README.md

```diff
@@ -4,39 +4,49 @@
 
 ## Contents
 
-- `library`: Ansible modules files for KubeVirt management
-- `tests`: Ansible playbooks to test KubeVirt modules
+- `lib`: Ansible modules files for KubeVirt management
+- `tests`: Ansible playbook examples and unit tests
 
 ## Requirements
 
-- Ansible 2.4.3.0
-- Kubernetes Python Module
+- Ansible >= 2.4.3.0
+- [KubeVirt Python SDK](https://github.com/kubevirt/client-python)
 - [KubeVirt](https://github.com/kubevirt/kubevirt)
 
 ## Installation and usage
 
-Use the Galaxy client to install the role:
+* Using Git
```

Comment:
`Using Git` sounds like there are other ways to install it. What about a numerated list?
1. Clone the repo: <...>
2. Install KubeVirt Python SDK
3. Add a playbook: <...>